utils/java/mvn.go

```diff
@@ -21,7 +22,7 @@ const (
 	mavenDepTreeJarFile    = "maven-dep-tree.jar"
 	mavenDepTreeOutputFile = "mavendeptree.out"
 	// Changing this version also requires a change in MAVEN_DEP_TREE_VERSION within buildscripts/download_jars.sh
```

Comment:
Please also update the value in `buildscripts/download_jars.sh` and execute the script (run `./buildscripts/download_jars.sh`) to update the jar.